host/src/FunctionsNetHost/Program.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.CommandLine;
+using FunctionsNetHost.Grpc;
+
+namespace FunctionsNetHost
+{
+    internal class Program
+    {
+        static async Task Main(string[] args)
+        {
+            try
+            {
+                Logger.LogInfo("Starting FunctionsNetHost");
+
+                var workerStartupOptions = await GetStartupOptionsFromCmdLineArgs(args);
+
+                using var appLoader = AppLoader.Instance;
```

Comment:
Is there a reason we're making this static?

Reply:
Flagging to address this in the next iteration. Merging this version now.